### ISSUE TITLE
about val result

### ISSUE THREAD

**Issue by JCongLang** [brdav/refign#13]
I obtained validation results using the model I trained
![1684685162199](https://github.com/brdav/refign/assets/108472371/8ec00133-855d-4441-ae8c-87cbfb51273c)

Is this result normal?

**Comment by brdav** [comment]
Hello, could you provide more detail on your issue?

**Comment by JCongLang** [comment]
> Hello, could you provide more detail on your issue?

I trained Refign-HRDA* model for Cityscapes→Dark Zurich domain adaptation，and evaluated Refign on the Dark Zurich validation set and Nighttime Driving 。
As presented in the chart above，I got 0.4746 on Dark Zurich and 0.5699 on Nighttime Driving。
Is the result within error range？
Thank you！

**Comment by brdav** [comment]
It is on the lower end. But not that, on Dark Zurich, model performance is always much worse on the validation set compared to the test set.
I recommend you train 3 models with different seeds, and then evaluate the one with best validation set performance.

**Comment by JCongLang** [comment]
Thank you for your help！

**Comment by Jumponthemoon** [comment]
> > Hello, could you provide more detail on your issue?
> 
> I trained Refign-HRDA* model for Cityscapes→Dark Zurich domain adaptation，and evaluated Refign on the Dark Zurich validation set and Nighttime Driving 。 As presented in the chart above，I got 0.4746 on Dark Zurich and 0.5699 on Nighttime Driving。 Is the result within error range？ Thank you！

Hi， I'm wondering how you get the nighttime driving dataset, i can't find it. Would you give a hint or link for help? 
Much Apprectiated!

**Comment by JCongLang** [comment]
> > > Hello, could you provide more detail on your issue?
> > 
> > 
> > I trained Refign-HRDA* model for Cityscapes→Dark Zurich domain adaptation，and evaluated Refign on the Dark Zurich validation set and Nighttime Driving 。 As presented in the chart above，I got 0.4746 on Dark Zurich and 0.5699 on Nighttime Driving。 Is the result within error range？ Thank you！
> 
> Hi， I'm wondering how you get the nighttime driving dataset, i can't find it. Would you give a hint or link for help? Much Apprectiated!

maybe you can download from https://drive.google.com/file/d/111FC70cuIO0qbdcRAEL7jqhzIdRIaz8i/view?usp=sharing

**Comment by Jumponthemoon** [comment]
> > > > Hello, could you provide more detail on your issue?
> > > 
> > > 
> > > I trained Refign-HRDA* model for Cityscapes→Dark Zurich domain adaptation，and evaluated Refign on the Dark Zurich validation set and Nighttime Driving 。 As presented in the chart above，I got 0.4746 on Dark Zurich and 0.5699 on Nighttime Driving。 Is the result within error range？ Thank you！
> > 
> > 
> > Hi， I'm wondering how you get the nighttime driving dataset, i can't find it. Would you give a hint or link for help? Much Apprectiated!
> 
> maybe you can download from https://drive.google.com/file/d/111FC70cuIO0qbdcRAEL7jqhzIdRIaz8i/view?usp=sharing

Thank you so much.
